server/protocols/jmap/src/main/java/org/apache/james/jmap/model/SetError.java

```diff
@@ -102,18 +102,26 @@ public String getType() {
     }
 
     @Override
-    public boolean equals(Object obj) {
-        if (obj instanceof SetError) {
-            SetError other = (SetError) obj;
-            return Objects.equal(this.type, other.type)
-                && Objects.equal(this.description, other.description)
-                && Objects.equal(this.properties, other.properties);
-        }
-        return false;
+    public boolean equals(Object o) {
```

Comment:
a merge/rebase conflict of 2 concurrent branches that require SetError comparison (my branch has been stuck in the review pipeline for longer!). equals() method had no real reason to be replaced (except for resolving a merge conflict).
toString was used for unit testing purpose IIRC.